roles/nv_gpu_install_from_commit/files/operator_helper_image_builder.yml

```diff
@@ -14,47 +14,68 @@ spec:
   source:
     type: Dockerfile
     dockerfile: |
-      # stable/Dockerfile
-      #
-      # https://developers.redhat.com/blog/2019/08/14/best-practices-for-running-buildah-in-a-container/
+      # git/Dockerfile
+      # https://github.com/containers/buildah/blob/master/contrib/buildahimage/upstream/Dockerfile
       #
       # Build a Buildah container image from the latest
-      # stable version of Buildah on the Fedoras Updates System.
-      # https://bodhi.fedoraproject.org/updates/?search=buildah
+      # upstream version of Buildah on GitHub.
+      # https://github.com/containers/buildah
       # This image can be used to create a secured container
       # that runs safely with privileges within the container.
+      # The containers created by this image also come with a
+      # Buildah development environment in /root/buildah.
       #
-      # --> adapted to use podman instead of buildah
-      FROM quay.io/fedora/fedora:32-x86_64
-
-      # Don't include container-selinux and remove
-      # directories used by dnf that are just taking
-      # up space.
-      RUN yum -y install podman fuse-overlayfs; rm -rf /var/cache /var/log/dnf* /var/log/yum.*
-
+      FROM registry.fedoraproject.org/fedora:latest
+      ENV GOPATH=/root/buildah
+      
+      # Install the software required to build Buildah.
+      # Then create a directory and clone from the Buildah
+      # GitHub repository, make and install Buildah
+      # to the container.
+      # Finally remove the buildah directory and a few other packages
+      # that are needed for building but not running Buildah
+      RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install --enablerepo=updates-testing \
+           make \
+           golang \
+           bats \
+           btrfs-progs-devel \
+           device-mapper-devel \
+           glib2-devel \
+           gpgme-devel \
+           libassuan-devel \
+           libseccomp-devel \
+           git \
+           bzip2 \
+           xz \
+           go-md2man \
+           runc \
+           fuse-overlayfs \
+           fuse3 \
+           containers-common; \
+           mkdir /root/buildah; \
+           git clone https://github.com/containers/buildah /root/buildah/src/github.com/containers/buildah; \
+           cd /root/buildah/src/github.com/containers/buildah; \
+           make;\
+           make install;\
+           rm -rf /root/buildah/*; \
+           yum -y remove bats golang go-md2man; \
+           yum clean all;
```

Comment:
could you update the indentation to make it clearer what is `yum install` arguments, and what are the following commands?

```
       RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install --enablerepo=updates-testing \
```  
(the indentation should start by splitting with the 1st line)